app.R

```diff
@@ -0,0 +1,1268 @@
+                                     min = 2,
+                                     max = 8,
+                                     value = 5 ,
+                                     step = 1) ,
+
+                         sliderInput(inputId = "simulationsBar",
+                                     label = "Number of simulations",
+                                     min = 1,
+                                     max = 1000,
+                                     value = 5 ,
+                                     step = 1)
+    
+                   )
+              ),                 
+
+              ##### Captions based simulationsBar ----
+                         conditionalPanel(
+                         condition = "input.simulationsBar <= 50",
+                         textOutput("hint1")
+                          ),
+                         conditionalPanel(
+                         condition = "input.simulationsBar >= 50",
+                         textOutput("hint2")
+                         )
+              ),
+            
+               ##### Table and Plot on the right ----
+                  # h3("Table and Plot"),
+                  column(
+                    width = 8,
+                         align = "center",
+                         conditionalPanel(
+                           condition = "input.random == 'Null with equal probabilities'",
+                            tableOutput("equalValues"),
+                           
+                           conditionalPanel(
+                             condition = "input.random == 'Null with equal probabilities'",
+                             plotOutput(outputId = "equalPlot",
+                                        width = '90%',
+                                        click = "plot_click"),
+                             
+                             tableOutput("equalPlotClickedpoints"),
+  
+                             htmlOutput(
+                               outputId = "equalText",
+                               class = "text-center")
+                                          ))),
+
+                  column(
+                    width = 8,
+                         align = "center",
+                    
+                         conditionalPanel(
+                           condition = "input.random == 'Null with different probabilities'",
+                           tableOutput("diffValues"),
+                           
+                           plotOutput(outputId = "diffPlot",
+                                      width = "90%",
+                                      click = "plot_click"),
+                           
+                           tableOutput("diffPlotClickedpoints"),
+                           
+                            htmlOutput(
+                              outputId = "diffText",
+                              class = "text-center")
+                           )))
+        ),
```

Comment:
No button to go to explore page!

Reply:
Hi Shravani,
I had gone through the content of the Example page with Dr.Hatfield before and he asked me to delete it. I forgot for what the reason but maybe because the Explore page is more like an individual part in this app, which user use their own data to explore the concepts and are not closely related to the previous page.